include/dsAudio.h

```diff
@@ -794,7 +760,8 @@ dsError_t  dsSetGraphicEqualizerMode(intptr_t handle, int mode);
 /**
  * @brief Gets the supported MS12 audio profiles
  *
- * This function will get the list of supported MS12 audio profiles
+ * For sink devices, this function will get the list of supported MS12 audio profiles.
+ * For the source devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
```

Comment:
in dsAVDTypes.h for audioProfileList variable in structure dsMS12AudioProfileList_t. Add all supported profile string.
remove eg (e.g: "Music,Movie,Voice,Night") add all supported prifles

typedef struct _dsMS12AudioProfileList_t {
    char audioProfileList[MAX_PROFILE_LIST_BUFFER_LEN]; ///< buffer containing the list of comma separated audio profile names for menu list
                                                        ///< **(e.g: "Music,Movie,Voice,Night")**
    int audioProfileCount;                              ///< Total number of profiles
} dsMS12AudioProfileList_t;

Reply:
updated